website/docs/cdktf/develop-custom-constructs/construct-design.mdx

```diff
@@ -20,15 +20,15 @@ You can write a custom construct in any language CDKTF supports.
 
 You can write `Construct` classes and store them locally, either within your CDKTF application or in a separate, shared directory. We recommend storing constructs in a separate directory when you have multiple CDKTF applications in a monorepo setup.
 
-When you write constructs in the same language as your CDKTF application, you can use the same importing mechanisms you use with other applications. Refer to [Constructs](/cdktf/concepts/constructs) in the CDKTF Concepts documentation for more details about using constructs.
+When you write constructs in the same language as your CDKTF application, you can use the same importing mechanisms you use with other applications. Refer to [Constructs](/terraform/cdktf/concepts/constructs) in the CDKTF Concepts documentation for more details about using constructs.
 
 ### Creating Constructs for Multiple Languages
 
 You have two options for writing constructs for users to consume in multiple programming languages.
 
-The first option is writing your construct in Typescript and using [`jsii`](https://aws.github.io/jsii/) to translate it to other languages. This approach requires some setup and configuration. Refer to the [JSII getting started guide](https://aws.github.io/jsii/user-guides/lib-author/quick-start/set-up/) for details. You can also use the `cdktf-construct` projen template to get started more quickly. Refer to [Publish to Public or Private Registries](/develop-custom-constructs/publish-and-distribute#publish-to-public-or-private-registries) for details.
+The first option is writing your construct in Typescript and using [`jsii`](https://aws.github.io/jsii/) to translate it to other languages. This approach requires some setup and configuration. Refer to the [JSII getting started guide](https://aws.github.io/jsii/user-guides/lib-author/quick-start/set-up/) for details. You can also use the `cdktf-construct` projen template to get started more quickly. Refer to [Publish to Public or Private Registries](https://terraform.io/develop-custom-constructs/publish-and-distribute#publish-to-public-or-private-registries) for details.
```

Comment:
🔴 Another! `https://terraform.io/develop-custom-constructs/publish-and-distribute#publish-to-public-or-private-registries`

Reply:
fixed here: https://github.com/hashicorp/terraform-cdk/pull/2693